src/Access/Authentication.cpp

```diff
@@ -117,39 +111,11 @@ namespace
             && external_authenticators.checkKerberosCredentials(authentication_method.getKerberosRealm(), *gss_acceptor_context);
     }
 
-#if USE_SSL
-    std::vector<uint8_t> hmacSHA256(const std::vector<uint8_t>& key, const std::string& data)
-    {
-        unsigned int len = SHA256_DIGEST_LENGTH;
-        std::vector<uint8_t> result(len);
-        HMAC(
-            EVP_sha256(),
-            key.data(),
-            static_cast<Int32>(key.size()),
-            reinterpret_cast<const uint8_t*>(data.data()),
-            data.size(),
-            result.data(),
-            &len);
-        return result;
-    }
-
-    std::vector<uint8_t> sha256(const std::vector<uint8_t>& data)
-    {
-        std::vector<uint8_t> hash(SHA256_DIGEST_LENGTH);
-        SHA256_CTX sha256;
-        SHA256_Init(&sha256);
-        SHA256_Update(&sha256, data.data(), data.size());
-        SHA256_Final(hash.data(), &sha256);
-        return hash;
-    }
-
-#endif
-
     std::string computeScramSHA256ClientProof(const std::vector<uint8_t> & salted_password [[maybe_unused]], const std::string& auth_message [[maybe_unused]])
     {
 #if USE_SSL
         auto client_key = hmacSHA256(salted_password, "Client Key");
-        auto stored_key = sha256(client_key);
+        auto stored_key = encodeSHA256(client_key);
```

Comment:
Nit: "encode" doesn't sound right in the context of hash functions. Just `sha256` like before was okay.

Reply:
I renamed it to `encodeSHA256` for consistency with other usages of OpenSSL helpers.